### PR TITLE
Website typography updates

### PR DESCRIPTION
**Pull request by jasoncostello**
:wave: 

HashiCorp is rolling out some design tweaks to the OSS sites. This PR is part of a larger initiative to implement more consistent typography across all sites. I've set the home page headings in Klavika (font) and updated the logo. 

It looks like this:

screencapture-localhost-4567-1487360517815

And like this on mobile: 
screen shot 2017-02-17 at 12 12 32 pm

cc/ @captainill @sethvargo

### CODE REVIEW

File: website/source/assets/stylesheets/_buttons.scss [hunk 1]
```diff
@@ -8,7 +8,6 @@
   color: $black;
   @include transition(color .3s ease-in-out);
   //@include btn-shadow();
-  font-family: $font-family-blanc-reg;
```
Comment (sethvargo) [inline website/source/assets/stylesheets/_buttons.scss]: Thank you. I was not a fan of Blanc lol.

File: website/source/assets/stylesheets/_variables.scss [hunk 2]
```diff
@@ -51,8 +51,7 @@ $link-hover-color:      darken($link-color, 15%);
 // Typography
 // -------------------------
 $font-family-open-sans: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-family-blanc:     'Blanc-Light', 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$font-family-blanc-reg: 'Blanc-Regular', 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-klavika:   'klavika-web', 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (sethvargo) [inline website/source/assets/stylesheets/_variables.scss]: @jasoncostello if we are making sweeping changes across all the projects, we can put some of this in middleman-hashicorp too

File: website/source/assets/stylesheets/_buttons.scss (hunk 1 above)
Comment (jasoncostello) [inline website/source/assets/stylesheets/_buttons.scss]: haha, nuked :fire:

File: website/source/assets/stylesheets/_variables.scss (hunk 2 above)
Comment (jasoncostello) [inline website/source/assets/stylesheets/_variables.scss]: Yes. I'll ping you in Slack to sync up about this.

**Pull request merged**